src/session/sidebar/row.rs

```diff
@@ -120,6 +126,28 @@ impl Row {
         glib::Object::new(&[]).expect("Failed to create Row")
     }
 
+    fn setup_expressions(&self) {
+        let imp = self.imp();
+        let item_expression = Row::this_expression("item");
```

Comment:
Matter of taste but I think using `Self` inside an impl block is more meaningful.
```suggestion
        let item_expression = Self::this_expression("item");
```